### PR TITLE
Added an example test for to demonstrate options.setAcceptInsecureCerts(true);

### PR DESCRIPTION
**Pull request by ravigajul**
### **User description**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Types of changes

- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

___

### **PR Type**
Tests

___

### **Description**
- Added a new test method `setAcceptInsecureCerts` in `OptionsTest.java`.
- Configured `ChromeOptions` to accept insecure certificates.
- Included navigation to "https://selenium.dev" to demonstrate the option.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Tests
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>OptionsTest.java</strong><dd><code>Add test for accepting insecure certificates in ChromeOptions</code></dd></summary>
<hr>
      
examples/java/src/test/java/dev/selenium/drivers/OptionsTest.java

<li>Added a new test method <code>setAcceptInsecureCerts</code>.<br> <li> Configured <code>ChromeOptions</code> to accept insecure certificates.<br> <li> Included navigation to "https://selenium.dev" to demonstrate the <br>option.<br>

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1762/files#diff-3eb3ff6008510d0d30b7f1d43c8857bdf29fda42534f08399bccfe1cd323eab5">+13/-0</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

**Review by harsha509 — APPROVED** [review]
Thank you @ravigajul !

File: examples/java/src/test/java/dev/selenium/drivers/OptionsTest.java [hunk 1]
```diff
@@ -47,5 +47,18 @@ public void setPageLoadStrategyNone() {
       driver.quit();
     }
   }
+
+  @Test
+  public void setAcceptInsecureCerts() {
+    ChromeOptions chromeOptions = new ChromeOptions();
+    options.setAcceptInsecureCerts(true);
```
Comment (harsha509) [inline examples/java/src/test/java/dev/selenium/drivers/OptionsTest.java]: chromeOptions.setAcceptInsecureCerts(true);

**Review by harsha509 — CHANGES_REQUESTED** [review]
HI @ravigajul ,

Thank you for the PR. 

There is an issue in the code, could you Please review the comments for more details and push the necessary changes

Thanks,
Sri

File: examples/java/src/test/java/dev/selenium/drivers/OptionsTest.java (hunk 1 above)
Comment (ravigajul) [reply examples/java/src/test/java/dev/selenium/drivers/OptionsTest.java]: My bad. I have updated it to the right reference chromeOptions

**Review by harsha509 — APPROVED** [review]

**Comment by harsha509** [comment]
Congratulations on your first contribution to selenium Docs @ravigajul 🎉 🎉 🎉

**Comment by ravigajul** [comment]
Thank you for your support and guidance. Feels good. Hoping to contribute more.Sent from my iPhoneOn Jun 23, 2024, at 4:25 AM, Sri Harsha ***@***.***> wrote:﻿
Congratulations on your first contribution to selenium Docs @ravigajul 🎉 🎉 🎉

**Pull request merged**